pkgs/development/python-modules/gekko/default.nix

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This should live in `pkgs/development/python-modules` instead.

Reply:
Not sure what you mean specifically but hopefully its fixed

Reply:
nevermind, I was reviewing on mobile and misread the path of the file. :)